src/cli.js

```diff
@@ -394,7 +413,11 @@ export default async function cli(argv: Array<string>, exit: boolean = false) {
         workspace    run a pyarn command inside a specific workspace
         help         get help with pyarn commands
     `
-  });
+    },
+    {
+      '--': true
```

Comment:
What's this for out of curiosity?

Reply:
It's what enables parsing of `--` inside minimist https://github.com/substack/minimist#var-argv--parseargsargs-opts